tools/msys2_setup_buildenv_stage2_32_64.sh

```diff
@@ -0,0 +1,211 @@
+
+# == Execute ==
+
+# Notes:
+#   Nov 2023 openssl and zlib installed in the base dev toolchain
+#            psutil is a dependency of aqtinstall but fails to install via dependencies so explicitly install python-psutil
+
+if [[ $INSTALL_PACKAGES -eq 1 ]]; then
+  new_step "Installing packages"
+
+  pacman -S --noconfirm \
+  ${MINGW_PACKAGE_PREFIX}-cmake \
+  ${MINGW_PACKAGE_PREFIX}-python-pip \
+  ${MINGW_PACKAGE_PREFIX}-python-pillow \
+  ${MINGW_PACKAGE_PREFIX}-python-lz4 \
+  ${MINGW_PACKAGE_PREFIX}-python-psutil \
+  ${MINGW_PACKAGE_PREFIX}-libjpeg-turbo \
+  ${MINGW_PACKAGE_PREFIX}-libtiff \
+  ${MINGW_PACKAGE_PREFIX}-freetype \
+  ${MINGW_PACKAGE_PREFIX}-lcms2 \
+  ${MINGW_PACKAGE_PREFIX}-libwebp \
+  ${MINGW_PACKAGE_PREFIX}-openjpeg2 \
+  ${MINGW_PACKAGE_PREFIX}-libimagequant \
+  ${MINGW_PACKAGE_PREFIX}-libraqm \
+  ${MINGW_PACKAGE_PREFIX}-SDL2 \
+  ${MINGW_PACKAGE_PREFIX}-clang \
+  ${MINGW_PACKAGE_PREFIX}-nsis \
+  ${MINGW_PACKAGE_PREFIX}-dfu-util
+
+  end_step $? "pacman -S --noconfirm <packages>"
+
+  run_step "Upgrading pip" "python -m pip install -U pip"
+  run_step "Installing Python clang" "python -m pip install clang"
+  run_step "Installing Python setuptools and wheel" "python -m pip install setuptools wheel"
+  run_step "Installing Python jinja2" "python -m pip install jinja2"
+
+  # Nov 2023
+  # From https://github.com/miurahr/aqtinstall Install section
+  # When you want to use it on MSYS2/Mingw64 environment, you need to set environmental variable export SETUPTOOLS_USE_DISTUTILS=stdlib,
+  # because of setuptools package on mingw wrongly raise error VC6.0 is not supported
+  export SETUPTOOLS_USE_DISTUTILS=stdlib
+
+  # this will also install dependencies and see note above re psutil issue
+  run_step "Installing Python package aqtinstall" "python -m pip install aqtinstall"
+fi
+
+if [[ $INSTALL_QT -eq 1 ]]; then
+  # if modules use syntax -m MODULE [MODULE]
+  QT_MODULES=
+  # supported versions of dependencies
+  QT_TOOLS=
+  QT_HOST=windows
+  QT_TARGET=desktop
+  QT_ARCH=
+  QT_INSTALL_DIR=qt
+
+  new_step "Searching for available Qt ${QT_VERSION} architectures"
+
+  qtarchlist="$(python -m aqt list-qt ${QT_HOST} ${QT_TARGET} --arch ${QT_VERSION})"
+
+  IFS=' ' read -ra qtarcharray <<< ${qtarchlist}
+
+  for arch in "${qtarcharray[@]}"; do
+    if [[ "${arch}" == win${MSYSTEM:(-2)}_mingw* ]];then
+      QT_ARCH=${arch}
+      break
+    fi
+  done
+
+  if [[ -z "$QT_ARCH" ]]; then
+    end_step 1 "Unable to find suitable Qt architecture in Qt ${QT_VERSION} for ${MSYSTEM}"
+  else
+    end_step 0 ""
+  fi
+
+  run_step "Installing Qt ${QT_VERSION}" \
+  "python -m aqt install-qt --outputdir ${QT_INSTALL_DIR} ${QT_HOST} ${QT_TARGET} ${QT_VERSION} ${QT_ARCH} ${QT_MODULES}"
+fi
+
+if [[ $DOWNLOAD_ARM -eq 1 ]]; then
+  # Note: the version needs to be kept in sync with EdgeTX/build-edgetx/dev/Dockerfile
+
+  DOWNLOAD_FILE="gcc-arm-none-eabi-10-2020-q4-major-win32.exe"
```

Comment:
arm-gnu-toolchain-11.3.rel1-mingw-w64-i686-arm-none-eabi.exe

Reply:
No, the current entry is correct, as it matches the toolchain in use for all builds. 

https://github.com/EdgeTX/build-edgetx/blob/main/dev/Dockerfile#L75